cli/cmd/admin/start.go

```diff
@@ -49,6 +50,15 @@ type Config struct {
 	SigningKeyID           string        `split_words:"true"`
 }
 
+type EmailConfig struct {
+	SMTPHost     string `default:"smtp.sendgrid.net" split_words:"true"`
+	SMTPPort     int    `default:"587" split_words:"true"`
+	SMTPUsername string `default:"apiKey" split_words:"true"`
+	SMTPPassword string `split_words:"true"`
+	SenderEmail  string `default:"admin@rilldata.com" split_words:"true"`
+	SenderName   string `default:"Rill Admin" split_words:"true"`
+}
```

Comment:
1. Please move into the `Config` struct – it's very convenient to have a single object showing all external config, even if it's getting long
2. Remove the defaults, add them to `.env` instead (incl. password) and update it in 1password